src/utils/error/handler.js

```diff
@@ -2,17 +2,24 @@ const errors = require("./errors");
 const httpStatus = require("../httpStatus");
 
 module.exports = (err, req, res, next) => {
-  
-  const { status, message } = errors.getErrorInfo(
-    err.name || err.message
-  );
+  let statusMessage;
+  let status;
+  let message;
+  let retval = { status, message };
```

Comment:
I'm not sure what you are trying to do here

Reply:
This is me trying to use one data structure for the two cases and failing miserably